src/frontend/src/handler/create_table.rs

```diff
@@ -965,15 +973,16 @@ pub async fn handle_create_table(
             .into()),
         };
         let mut graph = build_graph(plan);
-        graph.parallelism =
-            if cdc_source.is_some() || CdcTableType::from_properties(&properties).can_backfill() {
-                Some(Parallelism { parallelism: 1 })
-            } else {
-                session
-                    .config()
-                    .get_streaming_parallelism()
-                    .map(|parallelism| Parallelism { parallelism })
-            };
+        graph.parallelism = if cdc_table_info.is_some()
+            || CdcTableType::from_properties(&properties).can_backfill()
+        {
+            Some(Parallelism { parallelism: 1 })
```

Comment:
I guess this might not be expected, as distributed table is desired? If you mean to enforce the `Chain` to be singleton, then it can be automatically derived as long as you mark the the "source job" as singleton.